### ISSUE TITLE
accuracy

### ISSUE THREAD

**Issue by chengyalong123** [MaxZanella/CLIP-LoRA#1]
First of all, thank you for generously providing the source code. 
I want to know why I executed python main.py --root_path /path/to/your/data --dataset dataset_name --seed 1 on the flowers dataset, but the results are very different.
The accuracy of 0 shots in the paper is 71.4 and 16 shots is 98.0. When reproduced, it is only 67.4 and 72.4.

**Comment by MaxZanella** [comment]
Hello,

Thanks for your interest in our project!

I realized that I accidentally set the weight decay value to 1e2 instead of the default PyTorch value of 1e-2. This mistake probably happened when I was cleaning up unnecessary arguments right before pushing the code. I’m really sorry for the inconvenience! The corrected value should make things work much better now.

As for the difference in zero-shot accuracy, the reported value uses the template "a photo of a {}, a type of flower," which tends to be more effective than "a photo of a {}." This approach is often used in other adapter few-shot methods, which is why we chose to report it this way (even though we only use 'a photo of a {}.' template for CLIP-LoRA) . You can find the commented template in the corresponding dataset file (datasets/oxford_flowers.py), as well as for some other datasets (e.g. DTD). I’ll improve the printed values to make it clearer.

Thanks again for your feedback!

**Comment by chengyalong123** [comment]
Thank you for your prompt reply

> Hello,
> 
> Thanks for your interest in our project!
> 
> I realized that I accidentally set the weight decay value to 1e2 instead of the default PyTorch value of 1e-2. This mistake probably happened when I was cleaning up unnecessary arguments right before pushing the code. I’m really sorry for the inconvenience! The corrected value should make things work much better now.
> 
> As for the difference in zero-shot accuracy, the reported value uses the template "a photo of a {}, a type of flower," which tends to be more effective than "a photo of a {}." This approach is often used in other adapter few-shot methods, which is why we chose to report it this way (even though we only use 'a photo of a {}.' template for CLIP-LoRA) . You can find the commented template in the corresponding dataset file (datasets/oxford_flowers.py), as well as for some other datasets (e.g. DTD). I’ll improve the printed values to make it clearer.
> 
> Thanks again for your feedback!
